src/Talis/Persona/Client/Base.php

```diff
@@ -255,6 +308,14 @@ protected function performRequest($url, array $opts)
             $httpConfig['headers']['Content-Type'] = 'application/x-www-form-urlencoded';
         }
 
+        $version = $this->getClientVersion();
+        $httpConfig['headers']['User-Agent'] = "{$this->config['userAgent']}" .
+            "persona-php-client/{$version}";
+        $httpConfig['headers']['X-Request-ID'] = $this->getRequestId();
+        $httpConfig['headers']['X-Client-Version'] = $version;
+        $httpConfig['headers']['X-Client-Language'] = 'php';
+        $httpConfig['headers']['X-Client-Consumer'] = $this->config['userAgent'];
```

Comment:
I think this is really going to help you add the custom tracking for New Relic in Persona. You could get this info from the UA string if you wanted, but passing them through as variables makes it easier to load straight through to NR. 👍